Sources/ScipioKit/DescriptionPackage.swift

```diff
@@ -78,7 +78,13 @@ struct DescriptionPackage {
         return workspace
     }
 
-    init(packageDirectory: AbsolutePath, mode: Runner.Mode) throws {
+    /// Make DescriptionPackage from a passed package directory
+    /// - Parameter packageDirectory: A path for the Swift package to build
+    /// - Parameter mode: A Scipio running mode
+    /// - Parameter onlyUseVersionsFromResolvedFile: A boolean value indicates disabling force updating of Package.resolved.
+    ///   If it is `true` Package.resolved never be updated
+    ///   instead the resolving will fail if the Package.resolved is mis-matched with the workspace.
```

Comment:
It hards to read as sentence since there is no , for if.

```suggestion
    ///   If it is `true`, Package.resolved never be updated.
    ///   Instead, the resolving will fail if the Package.resolved is mis-matched with the workspace.
```